ssh.go

```diff
@@ -617,20 +619,22 @@ func (c *SSH) Exec(cmd string, opts ...exec.Option) error { //nolint:cyclop
 		}
 	}()
 
-	gotErrors := false
+	var errors []string
 
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
 		outputScanner := bufio.NewScanner(stderr)
 
 		for outputScanner.Scan() {
-			gotErrors = true
-			execOpts.AddOutput(c.String(), "", outputScanner.Text()+"\n")
+			msg := outputScanner.Text()
+			if msg != "" {
+				errors = append(errors, err.Error())
```

Comment:
Is that the right error to add here? I'm somehow failing to see where this is coming from.

Reply:
no